app/templates/main/select.html

```diff
@@ -38,7 +38,7 @@
             </label>
             <select class="govuk-select" id="to-quarter" name="to-quarter">
                 {% for item in params['to-quarter'] %}
-                    <option value="{{item}}">{{item}}</option>
+                    <option value="{{item}}" selected>{{item}}</option>
```

Comment:
I think we want selected only on the item with the highest value, not all the items?

Reply:
As per https://jinja.palletsprojects.com/en/3.0.x/templates/#for should be able to access `loop.last` to access the last item and set the attribute on that.